docs/victoriametrics/stream-aggregation/README.md

```diff
@@ -322,603 +310,9 @@ have no such requirement.
 
 It's recommended to use [aggregation windows](#aggregation-windows) when aggregating histograms if you observe [accuracy issues](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4580).
 
-See [the list of aggregate output](#aggregation-outputs), which can be specified at `output` field.
+See [the list of aggregate output](https://docs.victoriametrics.com/victoriametrics/stream-aggregation/configuration#aggregation-outputs), which can be specified at `output` field.
 See also [histograms over input metrics](#histograms-over-input-metrics) and [quantiles over input metrics](#quantiles-over-input-metrics).
 
-# Configuration
-
-Stream aggregation can be configured via the following command-line flags:
-
-- `-streamAggr.config` at [single-node VictoriaMetrics](https://docs.victoriametrics.com/victoriametrics/single-server-victoriametrics/)
-  and at [vmagent](https://docs.victoriametrics.com/victoriametrics/vmagent/).
-- `-remoteWrite.streamAggr.config` at [vmagent](https://docs.victoriametrics.com/victoriametrics/vmagent/) only. This flag can be specified individually
-  per each `-remoteWrite.url`, so the aggregation happens independently per each remote storage destination.
-  This allows writing different aggregates to different remote storage systems.
-
-These flags must point to a file containing [stream aggregation config](#stream-aggregation-config).
-The file may contain `%{ENV_VAR}` placeholders which are substituted by the corresponding `ENV_VAR` environment variable values.
-
-By default, the following data is written to the storage when stream aggregation is enabled:
-
-- the aggregated samples;
-- the raw input samples, which didn't match any `match` option in the provided [config](#stream-aggregation-config).
-
-This behaviour can be changed via the following command-line flags:
-
-- `-streamAggr.keepInput` at [single-node VictoriaMetrics](https://docs.victoriametrics.com/victoriametrics/single-server-victoriametrics/) 
-  and [vmagent](https://docs.victoriametrics.com/victoriametrics/vmagent/). At [vmagent](https://docs.victoriametrics.com/victoriametrics/vmagent/)
-  `-remoteWrite.streamAggr.keepInput` flag can be specified individually per each `-remoteWrite.url`.
-  If one of these flags is set, then all the input samples are written to the storage alongside the aggregated samples.
-- `-streamAggr.dropInput` at [single-node VictoriaMetrics](https://docs.victoriametrics.com/victoriametrics/single-server-victoriametrics/) 
-  and [vmagent](https://docs.victoriametrics.com/victoriametrics/vmagent/). At [vmagent](https://docs.victoriametrics.com/victoriametrics/vmagent/)
-  `-remoteWrite.streamAggr.dropInput` flag can be specified individually per each `-remoteWrite.url`.
-  If one of these flags are set, then all the input samples are dropped, while only the aggregated samples are written to the storage.
-
-## Aggregation outputs
```

Comment:
I think we should preseve this and other similar anchors.

Reply:
Updated in https://github.com/VictoriaMetrics/VictoriaMetrics/pull/9500/commits/ba25a821595069addb32faa35d1efa1e1b70ea3a